src/components/ec-btn/ec-btn.spec.js

```diff
@@ -0,0 +1,200 @@
+import EcBtn from './ec-btn.vue';
+
+describe('EcBtn', () => {
+  function mountBtn(props, mountOpts) {
+    const localVue = createLocalVue();
+
+    return mount(EcBtn, {
+      localVue,
+      propsData: { ...props },
+      ...mountOpts,
+    });
+  }
+
+  describe(':props', () => {
+    it('should render a <button> element by default', () => {
+      const wrapper = mountBtn();
+
+      expect(wrapper.find('.ec-btn').exists()).toBe(true);
+      expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it('should render a <button> element with class ".ec-btn--sm" when size is set to "sm"', () => {
+      const wrapper = mountBtn({
+        size: 'sm',
+      });
+
+      expect(wrapper.find('.ec-btn--md').exists()).toBe(false);
+      expect(wrapper.find('.ec-btn--sm').exists()).toBe(true);
+      expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it('should render a <button> element with class ".ec-btn--md" when size is set to "md"', () => {
+      const wrapper = mountBtn({
+        size: 'md',
+      });
+
+      expect(wrapper.find('.ec-btn--sm').exists()).toBe(false);
+      expect(wrapper.find('.ec-btn--md').exists()).toBe(true);
+      expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it('should render an <a> element when we define the "href" prop', () => {
+      const wrapper = mountBtn({
+        href: 'https://ebury.com/',
+      });
+
+      expect(wrapper.attributes().href).toBe('https://ebury.com/');
+      expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it('should render a router link when we define the "to" prop', () => {
+      const wrapper = mountBtn({
+        to: 'trade-finance',
+      },
+      {
+        stubs: ['router-link'],
+      });
+
+      expect(wrapper.attributes().to).toBe('trade-finance');
+      expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it('should render a router link with a custom tag when we define the "tag option"', () => {
+      const wrapper = mountBtn({
+        to: 'trade-finance',
+        tag: 'ebury',
+      },
+      {
+        stubs: ['ebury'],
+      });
+
+      expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it('should render a disabled button when disabled is set to true', () => {
+      const wrapper = mountBtn({
+        disabled: true,
+      });
+
+      expect(wrapper.attributes().disabled).toBe('disabled');
+      expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it('should render a button with an icon when the icon prop is defined', () => {
+      const wrapper = mountBtn({
+        icon: 'simple-check',
+      });
+
+      expect(wrapper.find('.ec-btn__icon').exists()).toBe(true);
+      expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it('should render a button with an outline when "outline" prop is set to true', () => {
+      const wrapper = mountBtn({
+        outline: true,
+      });
+
+      expect(wrapper.find('.ec-btn--outline').exists()).toBe(true);
```

Comment:
`expect(wrapper.classes('.ec-btn--outline')).toBe(true);`

please update it everywhere in this spec file.